scipy/stats/_continuous_distns.py

```diff
@@ -4701,6 +4739,8 @@ def _shape_info(self):
 
     def _fitstart(self, data):
         # Arbitrary, but the default a=b=1 is not valid
+        if isinstance(data, CensoredData):
+            data = data._uncensor()
         return super()._fitstart(data, args=(1, 0.5))
```

Comment:
Why is this needed? Doesn't the `super()._fitstart` (well, technically `_fit_loc_scale_support`) uncensor as needed?

If this code is really needed, it looks like it needs to be in `truncweibull_min_gen`, `genhyperbolic_gen`, `invweibull_gen`, `crystalball_gen`, and `studentized_range_gen`.

I added it to `pareto._fitstart`.

Reply:
You're right, this one was not needed. Fixed in https://github.com/scipy/scipy/pull/13699/commits/1c5db323199a6d593ecb84ca12762d2f26ff97bf